articles/azure-functions/functions-reference-powershell.md

```diff
@@ -0,0 +1,452 @@
+----                           -----
+MyQueue                        myData
+
+```
+
+## Logging
+
+Logging in PowerShell functions works similarly to regular PowerShell logging. You can use the logging cmdlets to write to each output stream. Each cmdlet maps to a log level that Functions uses.
+
+| Method              | Functions LogLevel | Description                            |
+|---------------------|--------------------|----------------------------------------|
+| `Write-Error`       | Error              | Writes to _Error_ level logging.       |
+| `Write-Warning`     | Warning            | Writes to _Warning_ level logging.     |
+| `Write-Information` | Information        | Writes to _Information_ level logging. |
+| `Write-Host`        | Information        | Writes to _Information_ level logging. |
+| `Write-Verbose`     | Trace              | Writes to _Trace_ level logging.       |
+| `Write-Debug`       | Debug              | Writes to _Debug_ level logging.       |
+
+[!Note]
+Verbose maps to the Functions log level `Trace`.
+
+In addition to these cmdlets, anything written to the pipeline will be written to the `Information` log level.
+
+You can [configure the trace-level threshold for logging](#configure-the-trace-level-for-console-logging) in the host.json file. For more information on writing logs, see [writing trace outputs](#writing-trace-output-to-the-console) below.
+
+Read [monitoring Azure Functions](functions-monitoring.md) to learn more about viewing and querying function logs.
+
+### Configuring the log level for a Function App
+
+Functions lets you define the threshold level for writing to the logs, which makes it easy to control the way logs are written to the console from your function. To set the threshold for all traces written to the console, use the `logging.logLevel.default` property in the `host.json` file. This setting applies to all functions in your function app.
+
+The following example sets the threshold to enable verbose logging for all functions, but sets the threshold to enable debug logging for a single function:
+
+```json
+{
+    "logging": {
+        "logLevel": {
+            "Function.MyFunction": "Debug",
+            "default": "Trace"
+        }
+    }
+}  
+```
+
+For more information, see [host.json reference](functions-host-json.md).
+
+## HTTP triggers and bindings
+
+HTTP and webhook triggers and HTTP output bindings use request and response objects to represent the HTTP messaging.  
+
+### Request object
+
+The Request object that's passed into the script comes from a type called `HttpRequestContext` which has the following properties:
+
+| Property  | Description                                                    | Type                      |
+|-----------|----------------------------------------------------------------|---------------------------|
+| _Body_    | An object that contains the body of the request.               | object*                   |
```

Comment:
Since you have the note below, you don't need the asterisk.  Makes it look like a pointer :)
```suggestion
| _Body_    | An object that contains the body of the request.               | object                    |
```